metaseq/data/streaming_src_tgt_dataset.py

```diff
@@ -123,6 +123,11 @@ def yield_src_tgt_blocks(iterable, block_size, drop_last, padding_idx):
     cur_tgt_block = []
     cur_block_remain = block_size
     for idx, (src, tgt) in enumerate(iterable):
+        if isinstance(
+            src, tuple
+        ):  # Adding negative examples not supported in this mode
```

Comment:
I'm not sure what's going on here. Can we add some comments explaining why we might get tuples?


Nit: move the comment inside the if block

Reply:
updated.